wikimedia/wikipedia-ios
# Import Share Reading Lists - address some TODOs

**Phabricator:** 
https://phabricator.wikimedia.org/T322158

### Notes
This PR addresses a couple of items from the Phab task:

1. Removed the payload title/description pre-population and instead added a default pre-population title of "My Reading List".
2. Saving a UserDefaults flag if the user tapped through to "Take survey", so that they never see the survey prompt again after pushing through to the survey.
3. Some slight cleanup 

### Test Steps
1. Fresh install and run the app. Tap a test link to go to the landing page (or deep link) and trigger the modal: https://test.wikipedia.org/wiki/Special:ReadingLists?limport=eyJsaXN0Ijp7ImVuIjpbMjE4NjksMzI3NDUsNDQ0NjksNDQ0NzQsODI3ODBdfX0=
2. Confirm the title textfield now pre-populates with "My Reading List"
3. Tap through to Create List, land on the list detail screen. Wait 5 seconds for the survey prompt to appear.
4. Tap "Take survey"
5. Tap the shared reading list deep link again. Tap through to create list (note you may need to add an incrementing number to the default title since reading lists with duplicate titles are not allowed).
6. Confirm the prompt no longer shows.
7. Terminate app. Repeat steps 1-3. Confirm survey prompt still no longer shows.

### Screenshots/Videos
